web/models/attendance/attendance.ex

```diff
@@ -14,9 +17,32 @@ defmodule CoursePlanner.Attendance do
 
   def changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:attendance_type, :class_id, :student_id])
+    |> cast(params, [:attendance_type, :class_id, :student_id, :comment])
     |> validate_required([:attendance_type, :class_id, :student_id])
     |> cast_assoc(:class)
     |> cast_assoc(:student)
+    |> validate_comment()
```

Comment:
is it possible to use `Ecto.Cahngeset.validate_inclusion` instead of creating a new function?